.github/workflows/cron.yml

```diff
@@ -17,14 +17,17 @@ jobs:
 
       - name: Update
         run: |
+          rm -rf *.json
           curl -H "Authorization: token ${{ secrets.GITHUB_TOKEN }}" \
-            https://api.github.com/repos/${{ secrets.REPOSITORY }}/releases > releases.json
+            https://api.github.com/repos/fishfolk/jumpy/releases > jumpy.json
+          curl -H "Authorization: token ${{ secrets.GITHUB_TOKEN }}" \
+            https://api.github.com/repos/fishfolk/punchy/releases > punchy.json
```

Comment:
Let's have a loop here in case we add more games in the future:

```sh
for game in "jumpy" "punchy"; do
  curl -H "Authorization: token ${{ secrets.GITHUB_TOKEN }}" \
    "https://api.github.com/repos/fishfolk/${game}/releases" > "${game}.json"
done
```